### PR TITLE
Ports Antagonist tips from Hippie Station

### PR DESCRIPTION
**Pull request by Cenrus**
https://github.com/HippieStation/HippieStation/pull/12023

Teaches new players how to play wizard, nukies, traitor and abductor. Far better than our current system, and actually functions as a tutorial. I'll add more tutorials for each antagonist in the future.

:cl: Steamp0rt and Lunaman
add: Improved antagonist tutorial
/:cl:

### CODE REVIEW

**Comment by Cenrus** [comment]
Needs review, I want to make sure the code is fine

File: code/modules/antagonists/_common/antag_popup.dm [hunk 1]
```diff
@@ -1,115 +0,0 @@
...
-
-/datum/antagonist
-	var/wikilink
-	var/popup_title = "You are an antagonist!"
-/datum/antagonist/abductor
-	wikilink = "https://tgstation13.org/wiki/Abductor"
-	popup_title = "You are an abductor!"
-/datum/antagonist/blob
-	wikilink = "https://tgstation13.org/wiki/Blob"
-	popup_title = "You are a blob!"
```
Comment (MarkSuckerberg) [inline code/modules/antagonists/_common/antag_popup.dm:22]: Some of these should be either left, or new pages need to be added as not all bee antags have pages, and although some don't really need individual pages, others such as blob do. What was here already at least includes a wiki page, which is better than nothing.

File: html/antagtips/abductor.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<html>
+	<center>
+	<h1>You are an Abductor!</h1>
+	<img src="ayylmao.png"/>
+	<p>You and your teammate have been chosen by the mothership to go and capture some humans.</p>
+	<p>There's two of you: the <b>scientist</b>, who must oversee the camera and surgery, and the <b>agent</b>, who must go down on the station to stun and cuff people.</p>
+	<p><img src="scitool.png"/>The scientist must utilize his <b>console</b> and <b>science tool</b>, set to marking mode, to send down the agent, go down himself, mark the target, retrieve it, mark the agent through the console and retrieve him too.</p>
+	<p><img src="alienorgan.png"/>After that, he must do the <b>experimentation surgery</b>.</p>
+	<p><img src="abaton.png"/>The agent must be sent down by the scientist to stun and cuff a target and drag it into a hidden spot and await the scientist.</p>
+	<p>People wearing <b>tinfoil hats</b> are immune to abductor batons.</p>
```
Comment (MarkSuckerberg) [inline html/antagtips/abductor.html:10]: Would be nice to include a wiki page link here in my opinion

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
Some things I think need addressed

**Review by ike709 — APPROVED** [review]

**Comment by Crossedfall** [comment]
Test merged onto Golden

**Review by Crossedfall — APPROVED** [review]
Worked within the test merge

**Pull request merged**